subprojects/version-control/src/test/groovy/org/gradle/vcs/git/internal/GitVersionControlSystemSpec.groovy

```diff
@@ -138,4 +147,22 @@ class GitVersionControlSystemSpec extends Specification {
         then:
         thrown GradleException
     }
+
+    def "can get versions"() {
```

Comment:
no pun? :)

Reply:
You want it to be called `can git versions`?